cmd/upgrade.go

```diff
@@ -206,5 +206,26 @@ func (uc *upgradeCmd) run(cmd *cobra.Command, args []string) error {
 		log.Fatalf("Error upgrading cluster: %s \n", err.Error())
 	}
 
+	apiloader := &api.Apiloader{
+		Translator: &i18n.Translator{
+			Locale: uc.locale,
+		},
+	}
+	b, e := apiloader.SerializeContainerService(uc.containerService, uc.apiVersion)
+
+	if e != nil {
+		return e
+	}
+
+	f := acsengine.FileSaver{
+		Translator: &i18n.Translator{
+			Locale: uc.locale,
+		},
+	}
+
+	if e = f.SaveFile(uc.deploymentDirectory, "apimodel.json", b); e != nil {
```

Comment:
This is based on the method used to save the apimodel after running scale https://github.com/Azure/acs-engine/blob/97e82f5edabd3535faa47d59d49c3aa60f80c7aa/cmd/scale.go#L364